cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,4840 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e31453ef",
+   "metadata": {},
+   "source": [
+    "Next, we will train our model on the consensus labels obtained using majority vote to compute out-of-sample predicted probabilities. We will then use these `pred_probs` to generate more informed `consensus_labels` using Cleanlab's [CROWDLAB](link to something) algorithm. We them use these `consensus_labels` to train a better model that generates more accurate `pred_probs`. This process itterates until the `consensus_labels` have no more improvement.\n",
+    "\n",
+    "`train_model()` trains a `resnet18` image model using cross validation to get out-of-sample predicted probabilities on the whole dataset. The function can be replaced with a custom training algorithm.\n",
+    "\n",
+    "We use the [AutoGluon](https://auto.gluon.ai/) library to train powerful image classifiers (Residual Network or Swin Transformer), but this requires a GPU to run in a reasonable time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "370c66ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_type = \"swin_base_patch4_window7_224\" # You can also try with \"swin_base_patch4_window7_224\"\n",
+    "\n",
+    "# Load model arguments\n",
+    "train_args = {\n",
+    "    \"num_cv_folds\": 5, # decrease this value if you want code to run faster\n",
+    "    \"verbose\": 1, \n",
+    "    \"epochs\": 50, # decrease this value if you want code to run faster\n",
+    "    \"holdout_frac\": 0.2, \n",
+    "    \"time_limit\": 7200, # decrease this value if you want code to run faster\n",
+    "    \"random_state\": 123\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "7c27da16",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "modified configs(<old> != <new>): {\n",
+      "root.img_cls.model   resnet101 != swin_base_patch4_window7_224\n",
+      "root.misc.num_workers 4 != 64\n",
+      "root.misc.seed       42 != 376\n",
+      "root.train.batch_size 32 != 16\n",
+      "root.train.early_stop_patience -1 != 10\n",
+      "root.train.early_stop_baseline 0.0 != -inf\n",
+      "root.train.epochs    200 != 50\n",
+      "root.train.early_stop_max_value 1.0 != inf\n",
+      "}\n",
+      "Saved config to /datasets/ulyana/examples/cifar10-multiannotator/fe61c054/.trial_0/config.yaml\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "----\n",
+      "Running cross-validation for model: swin_base_patch4_window7_224\n",
+      "----\n",
+      "Running Cross-Validation on Split: 0\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Model swin_base_patch4_window7_224 created, param count:                                         86753474\n",
```

Comment:
Is there a way to not print this output? It takes up almost 80% of this notebook and I don't think that's ideal